cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/triplestore/CgmesModelTripleStore.java

```diff
@@ -157,36 +167,41 @@ public boolean isNodeBreaker() {
         // Only consider is node breaker if all models that have profile
         // EquipmentCore or EquipmentBoundary
         // also have EquipmentOperation or EquipmentBoundaryOperation
-        Map<String, Boolean> eqModelHasEquipmentOperationProfile = new HashMap<>();
-        for (PropertyBag mp : r) {
+        Map<String, Boolean> modelHasOperationProfile = computeEqModelHasEquipmentOperationProfile(r);
+        boolean consideredNodeBreaker = modelHasOperationProfile.values().stream().allMatch(Boolean::valueOf);
+        if (consideredNodeBreaker) {
+            LOG.info(
+                    "All FullModel objects have EquipmentOperation profile, so conversion will be considered node-breaker");
+        } else {
+            LOG.info(
+                    "Following FullModel objects do not have EquipmentOperation profile, so conversion will not be considered node-breaker:");
+            modelHasOperationProfile.entrySet().forEach(meqop -> {
+                if (!meqop.getValue()) {
+                    LOG.info("    {}", meqop.getKey());
+                }
+            });
+        }
```

Comment:
Maybe you should check if `LOG.isInfoEnabled()` before entering this code, otherwise you might go through it for no reason...

Reply:
Done, and moved the log to a separate method.